KarhooSDK/Api/Response/CancelledByPayer.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CancelledByPayer.swift
+//  KarhooSDK
+//
+//  Created by Nurseda Balcioglu on 08/12/2020.
+//  Copyright © 2020 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+public struct CancelledByPayer: KarhooCodableModel {
+    
+    public let id: String
```

Comment:
I assume this is the User object. Should we not align the names with the backend or Android?

Reply:
Had to change the name as it was creating a conflict with a request file named Payer.